api/users/urls.py

```diff
@@ -13,5 +13,6 @@
     url(r'^(?P<user_id>\w+)/nodes/$', views.UserNodes.as_view(), name=views.UserNodes.view_name),
     url(r'^(?P<user_id>\w+)/preprints/$', views.UserPreprints.as_view(), name=views.UserPreprints.view_name),
     url(r'^(?P<user_id>\w+)/registrations/$', views.UserRegistrations.as_view(), name=views.UserRegistrations.view_name),
+    url(r'^(?P<user_id>\w+)/files/$', views.UserQuickFiles.as_view(), name=views.UserQuickFiles.view_name),
```

Comment:
OH. Ok I think I know what you were saying. Yes, there should be related. My bad!